goodkey/good_key.go

```diff
@@ -136,7 +135,7 @@ func (policy *KeyPolicy) GoodKey(ctx context.Context, key crypto.PublicKey) erro
 	// that has been administratively blocked.
 	if policy.blockedList != nil {
 		if blocked, err := policy.blockedList.blocked(key); err != nil {
-			return berrors.InternalServerError("error checking blocklist for key: %v", key)
+			return fmt.Errorf("error checking blocklist for key: %s", key)
```

Comment:
I don't think crypto.PublicKey has a .String() method, so isn't %v more appropriate than %s in this case?

Reply:
Oh whoops, you're definitely right.